extlinks/aggregates/management/commands/fill_user_aggregates.py

```diff
@@ -152,21 +154,16 @@ def _fill_user_aggregates(self, link_events, collection):
         None
         """
         for link_event in link_events:
-            if UserAggregate.objects.filter(
+            user_aggregate = UserAggregate.objects.filter(
                 organisation=collection.organisation,
                 collection=collection,
                 username=link_event["username__username"],
                 full_date=link_event["timestamp_date"],
                 on_user_list=link_event["on_user_list"],
-            ).exists():
+            )
+            if user_aggregate.exists():
                 # Query UserAggregate for the existing field
-                existing_link_aggregate = UserAggregate.objects.get(
-                    organisation=collection.organisation,
-                    collection=collection,
-                    username=link_event["username__username"],
-                    full_date=link_event["timestamp_date"],
-                    on_user_list=link_event["on_user_list"],
-                )
+                existing_link_aggregate = user_aggregate.first()
```

Comment:
same suggestion as in `fill_pageproject_aggregates.py`

Reply:
Done